docs: mention the need to override the coordinator policy hash (#867)
edgelesssys/contrast: docs/docs/components/policies.md

9. The Contrast Coordinator verifies that the started pod has a permitted policy hash in its `HOSTDATA` field.

After the last step, we know that the policy hasn't been tampered with and, thus, that the workload matches expectations and may receive mesh certificates.

## Platform Differences

Contrast uses different rules and data sections for different platforms.
This results in different policy hashes for different platforms.

The `generate` command automatically derives the correct set of rules and data sections from the `reference-values` flag.

> The `verify`, `set`, and `recover` commands need to know the coordinator's policy hash for in order to securely communicate with it.

Reviewer: ```suggestion
The `verify`, `set`, and `recover` commands need to know the coordinator's expected policy hash to verify its identity.
```
